Documentation/compatibility/clr_critical_section_spin_wait_removal.md

```diff
@@ -0,0 +1,38 @@
+## CLR critical section spin-wait removal
+
+### Scope
+
+Importance: Transparent
+Type: Runtime
+
+### Version Introduced
+
+4.8
+
+### Source Analyzer Status
+
+NotPlanned
+(not applicable)
+
+### Change Description
+
+The change is not expected to impact compatibility in the default mode (disabled).
```

Comment:
This should just be a change description, not a analysis of the issue. Something like the following:
```suggestion
By default, when a thread cannot acquire a critical section, it spin-waits until it can acquire the critical section. This behavior can lead to poor scalability.
Starting with applications running under .NET Framework 4.8, it is possible to disable the spin-waiting. This is an opt-in behavior; spin-waiting when a critical section cannot be acquired is the default behavior on all versions of the .NET Framework.
```